Mathlib/Order/SuccPred/LinearLocallyFinite.lean

```diff
@@ -145,8 +142,17 @@ instance (priority := 100) LinearLocallyFiniteOrder.isSuccArchimedean [LocallyFi
     have h_max : IsMax (succ^[n] i) := isMax_iterate_succ_of_eq_of_ne h_eq hnm.ne
     exact not_le.mpr (h_lt n) (h_max (h_lt n).le)
 
-instance (priority := 100) LinearOrder.isPredArchimedean_of_isSuccArchimedean [SuccOrder ι]
-    [PredOrder ι] [IsSuccArchimedean ι] : IsPredArchimedean ι where
+-- The `IsPredArchimedean` instance follows from the above and
+-- `isPredArchimedean_of_isSuccArchimedean`.
```

Comment:
It seems reasonable/preferrable to have this direct instance for symmetry, expectability and possibly performance

Reply:
I moved the stuff about `LinearOrder` to the top of the file so I didn't have to open the namespace twice. Honestly it'd probably be better to move it to the common file on `Is*Archimedean` once that exists.